common/v2/services/ApiService/Dex/Dex.ts

```diff
@@ -1,10 +1,15 @@
+import { ethers } from 'ethers';
+import { addHexPrefix } from 'ethereumjs-util';
+import BN from 'bn.js';
 import axios, { AxiosInstance } from 'axios';
 
 import { default as ApiService } from '../ApiService';
-import { TSymbol } from 'v2/types';
-import { DEXAG_PROXY_CONTRACT } from 'v2/config';
+import { TSymbol, ITxObject } from 'v2/types';
+import { DEXAG_MYC_TRADE_CONTRACT, DEXAG_MYC_HANDLER_CONTRACT } from 'v2/config';
 
-const DEX_BASE_URL = 'https://api.dex.ag/';
+import { DexTrade } from './types';
+
+const DEX_BASE_URL = 'https://api-v2.dex.ag/';
```

Comment:
Good candidate for `v2/config/constants.ts`?

Reply:
I was hesitant about this. Since it's an ApiService so only used once I figured it was good leave inside the service itself.